sound/soc/intel/boards/Kconfig

```diff
@@ -466,4 +466,18 @@ config SND_SOC_INTEL_TGL_RT1308_MACH
 
 endif ## SND_SOC_SOF_TIGERLAKE
 
+config SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH
+	tristate "CML with RT1011 and RT5682 in I2S Mode"
+	depends on I2C && ACPI
+	depends on MFD_INTEL_LPSS || COMPILE_TEST
+        select SND_SOC_RT1011
+        select SND_SOC_RT5682
+        select SND_SOC_DMIC
+        select SND_SOC_HDAC_HDMI
```

Comment:
take 3: please make this configuration dependent on SOF_HDA_LINK if you unconditionally select HDMI.